content/swagger/resources/measurement_families/definitions/measurement_family.yaml

```diff
@@ -0,0 +1,81 @@
+type: object
+required: ["code"]
+properties:
+  code:
+    type: string
+    description: Measurement family code
+    x-immutable: true
+  labels:
+    type: object
+    description: Measurement family labels for each locale
+    x-validation-rules: The `localeCode` is the code of an existing and activated locale
+    default: {}
+    properties:
+      localeCode:
+        type: string
+        description: Measurement family label for the locale `localeCode`
+  standard_unit_code:
+    type: string
+    description: Unit code used as the standard unit for this measurement family
+  units:
+    type: array
+    description: Family units
+    items:
+      type: object
+      properties:
+        code:
+          type: string
+          description: Measure code
+          x-immutable: true
+        labels:
+          type: object
+          description: Unit labels for each locale
+          x-validation-rules: The `localeCode` is the code of an existing and activated locale
+          default: {}
+          properties:
+            localeCode:
+              type: string
+              description: Unit label for the locale `localeCode`
+        convert_from_standard:
```

Comment:
You need to specify each field. It's missing the "operator" and "value" description here.

Reply:
Done

Reply:
Done

Reply:
Still missing the description for the "operator" and "value" field.

Reply:
It's just after, no?
items:
            type: object
            properties:
              operator:
                type: string
                x-validation-rules: The operator is one of the following values
                enum: ['mul','div','add','sub']
              value:
                type: string

Reply:
Done in PATCH PR: https://github.com/akeneo/pim-api-docs/pull/358